### PR TITLE
Add the active-active-aap flag into the EP file

### PR DESCRIPTION
**Pull request by kentwu1**

### CODE REVIEW

File: opflexagent/utils/ep_managers/endpoint_file_manager.py [hunk 1]
```diff
@@ -388,6 +388,8 @@ def filter_cidr_aaps(aaps):
         if ips or ips_aap:
             mapping_dict['anycast-return-ip'] = sorted(ips + ips_aap)
 
+        if 'active_active_aap' in mapping:
```
Comment (tbachman) [inline opflexagent/utils/ep_managers/endpoint_file_manager.py]: Couldn't you just do mapping_dict['active-active-aap'] = mapping.get('active_active_aap', False)?

Comment (kentwu1) [reply opflexagent/utils/ep_managers/endpoint_file_manager.py]: sure I can do that np.

Comment (kentwu1) [reply opflexagent/utils/ep_managers/endpoint_file_manager.py]: Hmm looks like changing this actually messed up quite a few UTs......That's probably why in this file we can see lots of "if XXX in mapping:" patterns. I would just follow suit then if no objections.

Comment (tbachman) [reply opflexagent/utils/ep_managers/endpoint_file_manager.py]: That's fine - go ahead and revert to the original patch.

**Comment by kentwu1** [comment]
Pushed a revision to revert it back to the original patch.

**Comment by tbachman** [comment]
Thanks - please post the backports through stable/newton.

**Pull request merged**